### PR TITLE
Updated scheme in Example project and added EGApp files for easier rc testing.

### PR DESCRIPTION
**Pull request by brettfazio**

### CODE REVIEW

File: Demo/EarlGreyExample/EarlGreyApp.podspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Install in main app target using pod 'EarlGrey/App'
+Pod::Spec.new do |s|
+
+  s.name = "EarlGreyApp"
+  s.version = "2.0.1"
```
Comment (tirodkar) [inline Demo/EarlGreyExample/EarlGreyApp.podspec]: 0.1?

File: Demo/EarlGreyExample/EarlGreyApp.podspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  s.license = { :type => "Apache 2.0", :file => "LICENSE" }
+
+  s.source = { :http => 'file:' + __dir__ + '/EarlGreyApp.zip' }
+  s.vendored_frameworks = "AppFramework.framework"
+
+  s.pod_target_xcconfig = { "FRAMEWORK_SEARCH_PATHS" =>"$(inherited) $(PLATFORM_DIR)/Developer/Library/Frameworks",
+                              "ENABLE_BITCODE" => "NO" }
+
+  #s.frameworks = [ "CoreData", "CoreFoundation", "CoreGraphics", "Foundation", "IOKit", "QuartzCore", "UIKit", "XCTest"]
```
Comment (tirodkar) [inline Demo/EarlGreyExample/EarlGreyApp.podspec]: nix?

File: Demo/EarlGreyExample/Podfile [hunk 3]
```diff
@@ -19,5 +19,5 @@ end
 
 target 'EarlGreyExampleUITests' do
   #inherit! :search_paths
-  pod 'EarlGreyTest', :git => 'https://github.com/brettfazio/EarlGrey.git', :branch => 'recentclone'
+  pod 'EarlGreyTest', :git => 'https://github.com/google/EarlGrey.git', :branch => 'earlgrey2pd'
```
Comment (tirodkar) [inline Demo/EarlGreyExample/Podfile]: ? This is already pushed?

Comment (brettfazio) [reply Demo/EarlGreyExample/Podfile]: Just to "earlgrey2pd" for testing purposes, it needs to exist somewhere for people to test it. 

We are not going to merge "earlgrey2pd"

File: Demo/EarlGreyExample/EarlGreyApp.podspec (hunk 2 above)
Comment (brettfazio) [reply Demo/EarlGreyExample/EarlGreyApp.podspec]: nixed.

File: Demo/EarlGreyExample/EarlGreyApp.podspec (hunk 1 above)
Comment (brettfazio) [reply Demo/EarlGreyExample/EarlGreyApp.podspec]: For testing, removed.

**Pull request closed without merging**